src/main/webapp/WEB-INF/tags/duration.tag

```diff
@@ -0,0 +1,16 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+
+<%@attribute name="duration" type="java.time.Duration" required="true" %>
+
+<c:if test="${duration.negative}">-</c:if>
```

Comment:
Hier haben wir als noch ein Leerzeichen for der Zahl. Also "- 2 std" wäre hier ein "-2 std" schöner?

Reply:
Bin nicht sicher wie ich das hinbekomme ohne im [tag](https://github.com/synyx/urlaubsverwaltung/blob/368635cc3d17dfced63edf2d8cc1d5b3937bcc7d/src/main/webapp/WEB-INF/tags/duration.tag) alles in eine Zeile zu packen. Hast du eine Idee?

Reply:
Ja. Hab mal bisschen recherchiert.

```jsp
<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
<%@taglib prefix="spring" uri="http://www.springframework.org/tags" %>

<%@attribute name="duration" type="java.time.Duration" required="true" %>

<%@ tag trimDirectiveWhitespaces="true" %>

<c:if test="${duration.negative}">-</c:if>
<c:if test="${duration.abs().toHoursPart() > 0}">
    <c:out value="${duration.abs().toHoursPart()}"/> <spring:message code="overtime.data.hours.abbr"/>&nbsp;
</c:if>
<c:if test="${duration.abs().toMinutesPart() > 0}">
    <c:out value="${duration.abs().toMinutesPart()}"/> <spring:message code="overtime.data.minutes.abbr"/>
</c:if>
<c:if test="${duration.toMinutes() == 0}">
    <spring:message code="overtime.person.zero"/>
</c:if>
```

Reply:
Das schmeißt dann alles raus
![2021-02-08_10-30](https://user-images.githubusercontent.com/142107/107201020-c0a86780-69f8-11eb-9eec-e6a776eb700c.png)

Reply:
Ja klar deshalb &nbsp; an den Stellen die man haben möchte. Siehe
`    <c:out value="${duration.abs().toHoursPart()}"/> <spring:message code="overtime.data.hours.abbr"/>&nbsp;`

Reply:
fixed:
![2021-02-08_10-35](https://user-images.githubusercontent.com/142107/107201562-6fe53e80-69f9-11eb-8c2e-0e69af5e7ee0.png)